app/experimenter/experiments/migrations/0218_update_results_data_schema.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.15 on 2022-10-13 17:50
+
+from django.db import migrations
+
+
+def update_results_data_schema(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+    windows = ["daily", "weekly", "overall"]
+    for experiment in NimbusExperiment.objects.all():
+        data = experiment.results_data
+        if data:
```

Comment:
I keep getting bit by doing this and not being explicit about `if data is not None:`